integration_tests/src/tests/commit_post_test.rs

```diff
@@ -720,7 +720,7 @@ pub fn aggregate_one_precommit_expires_test(v: &dyn VM) {
     apply_ok(
         v,
         &worker,
-        &robust_addr,
+        &miner_addr,
```

Comment:
Why was this change necessary? It shouldn't be

Reply:
@anorth This is because the Test Runtime always expects the receiver address to be an ID Address.


https://github.com/filecoin-project/builtin-actors/blob/7500bab29e5b14ed433f19145f27c53034c6bead/test_vm/src/messaging.rs#L643

Reply:
Can we change the emitter field here to be just an `Address` instead ? We can then worry about what to do with it once we get to the integration tests.

Reply:
The event emitter must always be an ID. The runtime should always give the actor an ID address as `receiver()`. The problem is somewhere in our TestVM which should resolve an address to an ID address before making it available as the receiver (but not too soon, in case the actor doesn't yet exist and needs to be created on the fly to receive the message). It's important that we can support calls to non-ID addresses like this but then have everything still work.